app/src/pages/common/settingsPage/defectTypesTab/defectSubType.jsx

```diff
@@ -208,7 +208,7 @@ export class DefectSubType extends Component {
           <DefectSubTypeForm
             form={data.locator}
             initialValues={data}
-            onDelete={this.showDeleteConfirmationDialog}
+            onDelete={this.stopEditing}
```

Comment:
Now it isn't delete action (`onCancel` prop name will be better here). Could you please also rename corresponding stuff inside of `DefectSubTypeForm` to `cancel` instead of `delete`.